CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ set(PYBIND11_HEADERS
   include/pybind11/descr.h
   include/pybind11/options.h
   include/pybind11/eigen.h
+  include/pybind11/embedded.h
```

Comment:
Sure. Actually, looking at the whole thing again, perhaps `interpreter.h` would be most appropriate. Some of the smaller functions currently in there could be moved into the main headers since they can be used even without the interpreter: `py::globals()` can be move into `pytypes.h` and `py::main()` could become a static function of `py::module`, i.e. `py::module::main()` which would actually be clearer. Then the only thing left are the interpreter controls, thus `interpreter.h`. Similarly, the CMake target could be `pybind11::interpreter`. What do you think?